recipes/mod_application_initialization.rb

```diff
@@ -19,5 +19,8 @@
 #
 
 include_recipe 'iis'
+install_method = node['iis']['windows_feature_install_method']&.to_sym
```

Comment:
heres an example of where the node attribute is used and on line 24 where the tranform_feature_name method is used.

All the recipes starting with mod_ are the same